docs/devices/PCI_E.md

```diff
@@ -40,7 +44,6 @@ To write (`/set`) a value publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/
 If value equals `ON` state is ON, if `OFF` OFF.
 
 ### Buzzer feedback (binary)
-ON means no buzzer noise.
```

Comment:
What's the correct description for buzzer feedback?

Reply:
When it's enabled it basically just makes a noise whenever a control is changed. Perhaps something like the following would do?

Enable buzzer feedback. The buzzer will make a noise whenever an action is carried out via this device such as the power state changing, child lock being enabled etc.

Reply:
Sounds good, then also please make a PR to update it [here](https://github.com/Koenkk/zigbee-herdsman-converters/blob/41c2143627699db8cf35e1524ee120c7c7f1f825/src/devices/weten.ts#L28)

Reply:
Done. Linked PR at https://github.com/Koenkk/zigbee-herdsman-converters/pull/7058